packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/DropdownDemo/DropdownDemo.tsx

```diff
@@ -81,16 +82,18 @@ export class DropdownDemo extends React.Component<{}, DropdownState> {
         isCogOpen
       });
     };
-    this.onCogSelect = event => {
+    this.onCogSelect = () => {
```

Comment:
As a demo, it would be good to show what args can be are expected here

Reply:
Added argument back in